sourcegraph/learn
# Title casing of product name

### What should this PR do?
Updates Sourcegraph cloud to Sourcegraph Cloud to reflect naming conventions

### Why are we making this change?
[Thread with context](https://sourcegraph.slack.com/archives/C01KPE23H4M/p1635524151156500)
Resolves DEVED-245

### What are the acceptance criteria? 
Ensure that all instances of "Sourcegraph cloud" have been edited to read as "Sourcegraph Cloud" with no new errors taking place

### How should this PR be tested?
Check out the branch or click the link for the deploy preview and review the modified files

## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.
